src/show.jl

```diff
@@ -21,3 +21,42 @@ Base.show(io::IO, A::DiskGeoArray) = begin
     end
     print(io, "\n  From file: $(filename(A))")
 end
+
+function Base.show(io::IO, stack::AbstractGeoStack)
+    printstyled(io, "$(Base.typename(typeof(stack)))", color=:blue)
+    print(io, " with $(length(keys(stack))) field(s)")
+    applicable(filename, stack) && print(io, ": $(filename(stack))")
```

Comment:
I think better to use `isa DiskGeoStack`, `applicable` is a bit mysterious as to why/why not it would have a `filename` method

Reply:
Done.